index.js

```diff
@@ -10,11 +13,45 @@ const router = require('./app/router');
 const PORT = process.env.PORT || 4000
 
 
-app.use(router);
+const url = 'mongodb://localhost:27017'
+const dbName = 'monProjetMongo';
+
+MongoClient.connect(url, function(err, client) {
+  console.log("Connecté à MongoDB");
+  const db = client.db(dbName);
+  client.close();
```

Comment:
```suggestion
  const db = client.db(dbName);
  client.close();
```
Tu peux supprimer ces lignes